src/styles/blocks/card.scss

```diff
@@ -0,0 +1,108 @@
+.card {
+  display: flex;
+  flex-flow: row wrap;
+  width: $block-width;
+  margin: 0 auto;
+
+  &__product {
+    display: flex;
+    flex-direction: column;
+    width: $card-width;
+    background-color: #fff;
+    border: 1px solid #f3f3f3;
```

Comment:
Colors are better to be stored as variables

Reply:
fixed